README.md

```diff
@@ -35,127 +44,179 @@ See the [input section](#Inputs) for more info about the inputs.
     ignorePattern:
 ```
 
-# Inputs
-The inputs for the action. The inputs `output` and `ignorePattern` are action specific. The rest are mapped to the parameters of PSScriptAnalyzer.
-Every input is of type string. 
+## Inputs
+
+This sections describes the input values used by the action.
+
+The values `output` and `ignorePattern` are action specific. The rest are mapped to the parameters
+of `Invoke-ScriptAnalyzer` cmdlet. Every value is of type string.
+
+To provide an array value, enclose the array in single quotes. For example:
+`'"value1", "value2", ...'`.
+
+### path
```

Comment:
I can't recall off the top of my head… Does the MS Documentation Style Guide frown on using inline code blocks in section headers? If not, I think all of these third-level headings named for input keys would provide a clearer signal that that's what these are. Even if it is frowned on, I think that's more targeted to their own docs which doesn't handle the ToC generation as well as GitHub, so I think it could still be worth bending in this direction.

Reply:
Implemented your suggestion.